modules/ui/intro/intro.js

```diff
@@ -51,29 +51,29 @@ export function uiIntro(context, skipToRapid) {
 
 
   function intro(selection) {
-    fileFetcher.get('intro_graph')
-      .then(dataIntroGraph => {
+    fileFetcher.get('intro_rapid_graph')
+      .then(dataIntroRapidGraph => {
         // create entities for intro graph and localize names
-        for (let id in dataIntroGraph) {
-          if (!_introGraph[id]) {
-            _introGraph[id] = osmEntity(localize(dataIntroGraph[id]));
+        for (let id in dataIntroRapidGraph) {
+          if (!_rapidGraph[id]) {
+            _rapidGraph[id] = osmEntity(localize(dataIntroRapidGraph[id]));
           }
         }
-        selection.call(startIntro);
       })
       .catch(function() { /* ignore */ });
-      fileFetcher.get('intro_fb_graph')
```

Comment:
do we still have a `intro_fb_graph.json` file hanging in the codebase?

Reply:
We do! It was removed from a direct include, because the underlying file fetching is now managed by the fileFetcher module. So, no more need to file-include it explicitly, but we still need to load it with the fetcher.